lib/ansible/galaxy/collection.py

```diff
@@ -621,7 +621,7 @@ def _get_galaxy_yml(b_galaxy_yml_path):
 
 def _build_files_manifest(b_collection_path, namespace, name):
     # Contains tuple of (b_filename, only root) where 'only root' means to only ignore the file in the root dir
-    b_ignore_files = frozenset([(b'*.pyc', False), (b'*.retry', False),
+    b_ignore_files = frozenset([(b'*.pyc', False), (b'*.retry', False), (b'.git', True),
```

Comment:
If I remember how this worked in 2.9 the 2nd tuple entry controls whether the file is excluded from the root only or throughout the collection. I would have thought we would want the latter so this should be

```suggestion
    b_ignore_files = frozenset([(b'*.pyc', False), (b'*.retry', False), (b'.git', False),
```

Then again looking at devel the pattern is `.git` so this does act the same as devel.

Reply:
I originally set the boolean in question to `True` and wrote the tests accordingly. But when I ran those tests against a devel branch, they failed, so I changed this to `False` for the sake of consistency.

The tests I talk about wait in the https://github.com/ansible/ansible/pull/69345 PR.